edx_toggles/tests/test_setting_toggles.py

```diff
@@ -78,3 +79,62 @@ def test_deleted_instances_are_not_listed(self):
         toggles.SettingToggle("NAME1", default=False, module_name="module1")
         instances = toggles.SettingToggle.get_instances()
         self.assertEqual([], instances)
+
+
+class NestedToggleExtractionTests(TestCase):
+    """
+    Tests for deeply nested toggle extraction from report.py
+    """
+
+    def test_level_1_nesting(self):
+        """
+        Test that Level 1 nested boolean values are correctly extracted.
+
+        Level 1: CONFIG['simple_toggle']
+        """
+        # Level 1 nesting: CONFIG['simple_toggle']
```

Comment:
Removing unnecessary comments. This is clear from the test itself.
```suggestion
        """
        Test that Level 1 nested boolean values are correctly extracted.
        """
```